doc(release): prepare release notes for OSS 23.10 (#2961)
centreon/centreon-documentation: i18n/fr/docusaurus-plugin-content-docs/version-23.10/releases/centreon-os.mdx

## Centreon Web

### 23.10.6

Release date: `January 12, 2024`

<details open>
  <summary>Enhancements</summary>

> - [API] Added dashboard shares to dashboards list.

Reviewer: ```suggestion
- [API] Added dashboard shares to the dashboard list.
```

</details>

<details>
  <summary>Bug fixes</summary>

> - [Authentication] Fixed hardcoded email value that block SSO authentication with OpenId Connect when parameter is not email.

Reviewer: ```suggestion
- [Authentication] Fixed hardcoded email value that blocked SSO authentication with OpenId Connect when parameter was not an email.
```

- [Configuration] Fixed error message displayed in configuration generation after upgrading to 23.04 version.
- [Event Logs] Fixed period in CSV export.

> - [Install] Fixed installation when database user already exists.

Reviewer: ```suggestion
- [Installation] Fixed installation issue occurring when database user already existed.
```

> - [Installation] Fixed issue adding twice poller ID in Centreon gorgone configuration.

Reviewer: ```suggestion
- [Installation] Fixed issue adding twice poller ID in Centreon Gorgone configuration.
```

> - [My Account] Fixed account properties edition for local user only.

Reviewer: ```suggestion
- [My Account] Made the name, login and password fields read-only for non-local users so as to prevent issues where users who had edited these properties were locked out of the platform.
```

- [Resource Status] Fixed bad host redirection in the services panel.

> - [Resource Status] Fixed default sorting order from worse to best resources status.

Reviewer: ```suggestion
- [Resource Status] Fixed default sorting order in Resources status: from worst to best.
```

> - [Web] Fixed an issue while showing herited contacts/contact groups in notification tab.

Reviewer: ```suggestion
- [Web] Fixed an issue where inherited contacts/contact groups were not displayed in the notifications tab.
```
